walk-safe/cypress/integration/mainpage.spec.js

```diff
@@ -0,0 +1,59 @@
+describe('Show main view of walk-safe App', () => {
+
```

Comment:
Once we get our BE connected here is some documentation for intercepting GraphQL calls: https://docs.cypress.io/guides/testing-strategies/working-with-graphql#Alias-multiple-queries-or-mutations

Reply:
Thx Caroline - will dig into the docs then